lib/y2lan_restart_common.pm

```diff
@@ -149,14 +149,23 @@ sub check_network_status {
     }
     assert_script_run 'cat journal.log';                              # print journal.log
     my $is_empty = script_run('[ -s journal.log ]');
+    # if journal.log is empty implies that we could not find logs indicating that the services is not restarted or reloaded
     if ($is_empty) {
-        assert_equals($expected_status, 'no_restart', "Network restart didn't occur, even though it was expected.");
+        assert_equals($expected_status, 'no_restart', "Network service should have been restarted/reloaded. journalctl cannot find events");
     }
+    # if journal.log contains one of the shutting down|ifdown all|Stopping wicked then it is assumed that the service is restarted otherwise it is reloaded.
     else {
-        assert_equals($expected_status, 'restart', "Network restart occurred, even though it wasn't expected.");
+        my $journal_findings = script_output('cat journal.log');
+        my @matched          = grep { $_ =~ /(shutting down|ifdown all|Stopping wicked)/ } $journal_findings;
```

Comment:
taking into account my comment about comment :) I think it make sense to extract this query into global variable so you would have it in single place. Also you could mention variable in comment above instead exact query

Reply:
make sense. fixed